app/components/pipeline/Header/index.js

```diff
@@ -189,7 +174,7 @@ class Header extends React.Component {
     if (this.props.pipeline.description) {
       return <Emojify text={this.props.pipeline.description} />;
     } else {
-      return this.repositoryUrl();
+      return this.props.pipeline.repository.url;
```

Comment:
There's `repository.url` which is the raw value they put into the repository setting (which is what's printed under the name if it doesn't have a description), and then there's `repository.provider.url` which is generated from the Ruby's. It's used to detect the setup instructions too, to give the webhook setup screen. But none of that is affected by whether you setup webhooks or not… we don't know about their webhook setup really.